src/Arbitrary.hs

```diff
@@ -755,18 +754,18 @@ fishAvatars =
     , "tetraodontiformes/thumnails/tetraodon_nigroviridis.gif"
     ]
 
-mkFishUser :: (GenArbitrary r, ActionM r m) => Maybe SchoolClass -> URL -> m User
+mkFishUser :: (GenArbitrary m, ActionM m) => Maybe SchoolClass -> URL -> m User
 mkFishUser mSchoolClass (("http://zierfischverzeichnis.de/klassen/pisces/" <>) -> avatar) = do
     let first_last = cs . takeBaseName . cs $ avatar
         (fnam, lnam) = case ST.findIndex (== '_') first_last of
             Nothing -> error $ "mkFishUser: could not parse avatar url: " <> show avatar
             Just i -> ( UserFirstName $ ST.take i first_last
                       , UserLastName  $ ST.drop (i+1) first_last
                       )
-    role <- Student <$> maybe (persistent genArbitrary) pure mSchoolClass
+    role <- Student <$> maybe genArbitrary pure mSchoolClass
     let pu = ProtoUser Nothing fnam lnam role Nothing Nothing
     -- FIXME: change avatar in the database, not just in the user returned from this function!
-    (userAvatar .~ Just avatar) <$> currentUserAddDb addUser pu
+    (userAvatar ?~ avatar) <$> currentUserAddDb (AddUser (UserPassInitial "streng geheim!")) pu
```

Comment:
It's a silly joke.  I assumed it was harmless because it's obvious that any meaning is conincidental and unimportant.

What would you like me to use instead?